apps/docs/src/app/core/component-docs/file-uploader/examples/file-uploader-min-and-max-size-example/file-uploader-min-and-max-size-example.component.ts

```diff
@@ -9,13 +9,13 @@ export class FileUploaderMinAndMaxSizeExampleComponent {
     files: File[];
     invalidFiles: File[];
 
-    handleFileSection(): void {
-        alert('Files selected successfully!!!');
+    handleFileSection(files: File[]): void {
+        alert(files.length + ' Files selected successfully!!!');
     }
 
-    handleInvalidFiles($event): void {
-        alert(' Invalid file selection ');
-        this.invalidFiles = $event;
+    handleInvalidFiles(files: File[]): void {
+        alert(files.length + ' Invalid files selected ');
+        this.invalidFiles = files;
```

Comment:
same as my other comment about i18n. At least the application developer should be able to customize how they want to handle these 2 functions.

Reply:
You are not missing anything :) I know these are in the examples, but don't you think the application developers would like to handle the success/failure cases in a similar way? Like provide an API for this?